app/views/notification.js

```diff
@@ -0,0 +1,31 @@
+import Mn from 'backbone.marionette';
+import application from '../application';
+
+const Notification = Mn.ItemView.extend({
+
+    template: require('./templates/notification'),
+
+    ui: {
+        notification: '.notification',
+    },
+
+    initialize: function(options) {
+        this.message = options.message;
+    },
+
+    onRender: function() {
+        this.ui.notification.show();
+        this.ui.notification.addClass('show');
+        setTimeout(function() {
+            $('.notification').hide();
```

Comment:
It's better to keep visual effect (like animation, transitions, etc) to CSS instead of using jQuery methods. I suggest you set the attribute `aria-hidden` to `true`/`false`, and use a CSS rule that `[aria-hidden=true]{ display: none}`.